todoman/model.py

```diff
@@ -305,7 +304,7 @@ def color_rgb(self):
     def color_ansi(self):
         rv = self.color_rgb
         if rv:
-            return rgb8(*rv)
+            return '\33[38;2;{!s};{!s};{!s}m'.format(*rv)
```

Comment:
It's a shame that click doesn't support RGB yet. Let's leave this as-is for now then.